pkg/hostpath/hpv.go

```diff
@@ -1,22 +1,26 @@
 package hostpath
 
-import "sync"
+import (
+	"strconv"
+	"sync"
+)
 
 type hostPathVolume struct {
-	VolID          string     `json:"volID"`
-	VolName        string     `json:"volName"`
-	VolSize        int64      `json:"volSize"`
-	VolPath        string     `json:"volPath"`
-	VolAccessType  accessType `json:"volAccessType"`
-	TargetPath     string     `json:"targetPath"`
-	SharedDataKey  string     `json:"sharedDataKey"`
-	SharedDataKind string     `json:"sharedDataKind"`
-	SharedDataId   string     `json:"sharedDataId"`
-	PodNamespace   string     `json:"podNamespace"`
-	PodName        string     `json:"podName"`
-	PodUID         string     `json:"podUID"`
-	PodSA          string     `json:"podSA"`
-	Allowed        bool       `json:"allowed"`
+	VolID            string     `json:"volID"`
```

Comment:
Could we add doc-comments on those fields?

Reply:
sure I'll try to add that with the next round of readme updates I have for the blog post